docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '2'
```

Comment:
I think given that it's just a single service we don't really need a `docker-compose.yml` file. We can just use plain docker or wrap the commands above in a shell script.